### PR TITLE
azure openai add gpt-4-1106-preview、gpt-4-vision-preview models

### PR DESCRIPTION
**Pull request by charli117**

### CODE REVIEW

File: web/i18n/lang/common.en.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ const translation = {
         'How much to penalize new tokens based on their existing frequency in the text so far.\nDecreases the model\'s likelihood to repeat the same line verbatim.',
       max_tokens: 'Max token',
       max_tokensTip:
-        'Used to limit the maximum length of the reply, in tokens. \nLarger values may limit the space left for prompt words, chat logs, and Knowledge. \nIt is recommended to set it below two-thirds.',
+        'Used to limit the maximum length of the reply, in tokens. \nLarger values may limit the space left for prompt words, chat logs, and Knowledge. \nIt is recommended to set it below two-thirds\ngpt-4-1106-preview、gpt-4-vision-preview max token input 128k、output 4k.',
```
Comment (crazywoola) [inline web/i18n/lang/common.en.ts]:
```suggestion
        'Used to limit the maximum length of the reply, in tokens. \nLarger values may limit the space left for prompt words, chat logs, and Knowledge. \nIt is recommended to set it below two-thirds\ngpt-4-1106-preview, gpt-4-vision-preview max token (input 128k output 4k)',
```

File: web/i18n/lang/common.zh.ts [hunk 2]
```diff
@@ -65,7 +65,7 @@ const translation = {
         '影响常见与罕见词汇使用。\n值较大时，倾向于生成不常见的词汇和表达方式。\n值越小，更倾向于使用常见和普遍接受的词汇或短语。',
       max_tokens: '单次回复限制 max_tokens',
       max_tokensTip:
-        '用于限制回复的最大长度，以 token 为单位。\n较大的值可能会限制给提示词、聊天记录和知识库留出的空间。\n建议将其设置在三分之二以下。',
+        '用于限制回复的最大长度，以 token 为单位。\n较大的值可能会限制给提示词、聊天记录和知识库留出的空间。\n建议将其设置在三分之二以下。\ngpt-4-1106-preview、gpt-4-vision-preview上限 输入128k，输出4k。',
```
Comment (crazywoola) [inline web/i18n/lang/common.zh.ts]:
```suggestion
        '用于限制回复的最大长度，以 token 为单位。\n较大的值可能会限制给提示词、聊天记录和知识库留出的空间。\n建议将其设置在三分之二以下。\ngpt-4-1106-preview、gpt-4-vision-preview 最大长度 (输入128k，输出4k)',
```

**Review by crazywoola — CHANGES_REQUESTED** [review]

**Review by crazywoola — DISMISSED** [review]

**Review by crazywoola — DISMISSED** [review]
LGTM

**Review by takatost — APPROVED** [review]
thx!!

**Pull request merged**